visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -26,6 +29,12 @@ public class VisualClient : IDisposable
         public bool CaptureDom { get; set; } = false;
         private readonly ResiliencePipeline _retryPipeline;
 
+        public string? CurrentTestClass { get; set; }
+        public string? CurrentTestName { get; set; }
```

Comment:
If we move to the singleton instnace + we will run tests in parallel - it would be super hard to keep this API working correctly. 

We also don't have any reference to it - I think we should remove them, right?